src/app/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { bool, string, arrayOf, element, oneOf, shape } from 'prop-types';
+import styled from '@emotion/styled';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  C_CONSENT_BACKGROUND,
+  C_CONSENT_ACTION,
+  C_PEBBLE,
+  C_CONSENT_CONTENT,
+  C_WHITE,
+  C_EBON,
+  C_GHOST,
+} from '@bbc/psammead-styles/colours';
+import {
+  getDoublePica,
+  getLongPrimer,
+  getBodyCopy,
+} from '@bbc/gel-foundations/typography';
+import { getSansBold, getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_SPACING_DBL,
+  GEL_SPACING,
+  GEL_SPACING_QUAD,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+const MIN_TAP_HEIGHT = '2.75rem';
+
+const COMMON_HEADING_STYLES = `
+  color: ${C_WHITE};
+  margin-top: 0;
+  margin-bottom: 0;
+`;
+
+const Wrapper = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  background-color: ${C_CONSENT_BACKGROUND};
+  max-height: 100vh;
+  padding-right: ${GEL_MARGIN_BELOW_400PX};
+  padding-left: ${GEL_MARGIN_BELOW_400PX};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding-left: ${GEL_MARGIN_ABOVE_400PX};
+    padding-right: ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const BannerPage = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  max-height: 100vh;
```

Comment:
Ah does this stop the top of the banner going off the screen on a very small viewport size? Does that mean you get a vertical scroll to allow you to get to the bottom in this edge case: https://github.com/bbc/simorgh/pull/8718/files#diff-f4a0ed012b23d04494a87f7cc510a3a33910e55f02342109d01f34d5390b121aR61 This would be fine btw, I'm just making sure I understand.

Reply:
Yes, that's right. This is less an edge case and more a common case though.